api/services/AccessControlService/NewsAccessControl/allowClientToEditNews.ts

```diff
@@ -0,0 +1,6 @@
+import { addUserRoles } from '../operations';
+import { getNewsEditRole } from './getNewsRoles';
+
+export default async function allowClientToEditNews(clientId: number, newsId: number) {
+  return addUserRoles(clientId, await getNewsEditRole(newsId, false));
```

Comment:
> function getNewsEditRole(newsId: number, forceUpdate = false) { ... }

Now that the forceUpdate argument has a default value, and the default value is the same as your input, you can simply leave is as `await getNewsEditRole(newsId)` here.